sequins.go

```diff
@@ -150,7 +168,14 @@ func (s *sequins) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	key := strings.TrimPrefix(r.URL.Path, "/")
-	res, err := s.index.Get(key)
+
+	currentIndex := s.indexMonitor.AcquireRef()
+	if currentIndex == nil {
+		log.Fatal("Tried to make HTTP request before initializing the index") 
+	}
```

Comment:
If there're no versions to load from, it should error before it gets to this point (and it should error, because that's better than starting up a http server that always 404s, I think).